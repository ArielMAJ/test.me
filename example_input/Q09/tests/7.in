5
-1000002 -1000001 -1000000 -1000000 -1000000