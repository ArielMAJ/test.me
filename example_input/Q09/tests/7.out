2 -1000000
